tests/datasets/test_spacenet.py

```diff
@@ -339,6 +356,49 @@ def test_plot(self, dataset: SpaceNet5) -> None:
         plt.close()
 
 
+class TestSpaceNet6:
+    @pytest.fixture(params=["PAN", "RGBNIR", "PS-RGB", "PS-RGBNIR", "SAR-Intensity"])
+    def dataset(
+        self, request: SubRequest, monkeypatch: MonkeyPatch, tmp_path: Path
+    ) -> SpaceNet6:
+        radiant_mlhub = pytest.importorskip("radiant_mlhub", minversion="0.2.1")
```

Comment:
Should we bump this to the module level instead of the class level? Every class does the same thing right?

Reply:
Not sure how this would work. Each dataset has its own set of params and md5 hashes.

Reply:
I'm only talking about the `pytest.importorskip` line. This line can be defined at the function level, class level, or module level. So if you move this to the top of the file, it will be shared by all classes and functions.